.github/workflows/documentation.yml

```diff
@@ -0,0 +1,146 @@
+---
+name: Documentation
+
+# Builds Markdown documentation for core and addons.
+
+on:
+  push:
+    branches:
+      - main
+      - releasebranch_*
+  pull_request:
+
+permissions: {}
+
+jobs:
+  ubuntu:
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+      cancel-in-progress: true
+
+    runs-on: ubuntu-22.04
+    env:
+      PYTHONWARNINGS: always
+
+    steps:
+      - name: Checkout core
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: OSGeo/grass
+          path: grass
+
+      - name: Get dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y wget git gawk findutils
+          xargs -a <(awk '! /^ *(#|$)/' "grass/.github/workflows/apt.txt") -r -- \
+              sudo apt-get install -y --no-install-recommends --no-install-suggests
+
+      - name: Set up Python
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
+        with:
+          python-version: "3.12"
```

Comment:
Do we want to keep the version updated like python-code-quality.yml?

See 
https://github.com/OSGeo/grass/blob/c112752ef1e070f6d202d262211e4275f7671a83/.github/workflows/python-code-quality.yml#L28-L30

https://github.com/OSGeo/grass/blob/c112752ef1e070f6d202d262211e4275f7671a83/.github/workflows/python-code-quality.yml#L58-L62

Reply:
With the default behavior, I think it is enough, and you could remove `continue-on-error`, as it may hide real problems that the actions will report. 

Is the addons directory really missing, or just misspecified?

Reply:
Yes, updated. Done.